result/ospfv3lsa.yaml

```diff
@@ -0,0 +1,386 @@
+        address_prefix:
+          description: >-
+            The first IPv6 address prefix to be advertised in the LSA.
+          type: string
+          format: ipv6
+          x-field-uid: 2
+        prefix_length:
+          description: >-
+            The length of the IPv6 address prefix, in bits.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+        metric:
+          description: >-
+            The cost metric value for the route to this destination router.
+          type: integer
+          format: uint32
+          x-field-uid: 4
+
+    Ospfv3.LsaHeader:
+      description: >-
+        Attributes in LSA Header.
+      type: object
+      properties:
+        lsa_id:
+          description: >-
+            LSA ID in the IPv4 format. The Link State ID for the specified LSA type.
+          type: string
+          format: ipv4
+          x-field-uid: 1
+        advertising_router_id :
+          description: >-
+            The router ID (in the IPv4 format) of the router that originated the LSA.
+          type: string
+          format: ipv4
+          x-field-uid: 2
+        sequence_number:
+          description: >-
+            Sequence number to detect old and duplicate
+            LSAs. The greater the sequence number the more recent the LSA.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+        age:
+          description: >-
+            The time since the LSA's generation in seconds.
+          type: integer
+          format: uint32
+          x-field-uid: 4
+          
+    Ospfv3.Link:
+      description: >-
+        Generic attributes used to identify links within OSPFv3.
+      type: object
+      properties:
+        type:
+          description: >-
+            The data associated with the link type. The value is
+            dependent upon the subtype of the LSA.
+            - point_to_point: The LSA represents a point-to-point connection to another router.
+            - transit: The LSA represents a connection to a transit network.
+            - stub: The LSA represents a connection to a stub network.
+            - virtual: The LSA represents a virtual link connection.
+          type: string
+          x-enum:
+            point_to_point:
+              x-field-uid: 1
+            transit:
+              x-field-uid: 2
+            stub:
+              x-field-uid: 3
+            virtual:
+              x-field-uid: 4
+          x-field-uid: 1
+        metric:
+          description: |-
+            The data associated with the link type. The value is
+            dependent upon the subtype of the LSA. When the connection is
+            to a stub network it represents the mask; for p2p connections
+            that are unnumbered it represents the ifIndex value of the
+            router's interface; for all other connections it represents
+            the local system's IP address.
+          type: integer
+          format: uint32
```

Comment:
metric field has different data type between models-yang (uint16) and OTG model (uint32). Which one is correct? Both should be consistent.

Reply:
It should be uint32 - so need to change it in models-yang.